app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -538,26 +600,28 @@ const SendBitcoinDetailsScreen = ({
           <Text style={Styles.fieldTitleText}>{LL.SendBitcoinScreen.amount()}</Text>
           <View style={Styles.fieldBackground}>
             <View style={Styles.currencyInputContainer}>
-              {fromWallet.walletCurrency === WalletCurrency.Btc &&
-                paymentAmount.currency === WalletCurrency.Btc && (
+              {sendingWalletDescriptor.currency === WalletCurrency.Btc &&
+                paymentDetail.unitOfAccountAmount?.currency === WalletCurrency.Btc && (
                   <>
                     <FakeCurrencyInput
                       {...testProps("BTC Amount")}
-                      value={paymentAmountToDollarsOrSats(btcAmount)}
+                      value={paymentAmountToDollarsOrSats(
+                        paymentDetail.unitOfAccountAmount,
+                      )}
                       onChangeValue={setAmountsWithBtc}
                       prefix=""
                       delimiter=","
                       separator="."
                       precision={0}
                       suffix=" sats"
                       minValue={0}
-                      editable={!isFixedAmountInvoice}
+                      editable={Boolean(paymentDetail.setAmount)}
```

Comment:
this line feels weird. setAmount is a function and it's been converted to a boolean?

btw, I'm not a huge fan of a the Boolean requirement, maybe we could drop the eslint rules? (I guess here it's triggering a relevant question)

Reply:
Replacing it with `paymentDetail.canSetAmount`